releases/release-1.23/release-notes/maps/pr-105035-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 105035
+releasenote:
+  text: migrated pkg/proxy/winuserspace to structured logging
```

Comment:
Migrated?

Reply:
Perhaps we should have consistent wording around release notes regarding migration to structured logging
e.g. this one and the one that relates to this [comment](https://github.com/kubernetes/sig-release/pull/1718#discussion_r722504299)
structured logging vs structured logs

Reply:
FWIW in 1.21 we aggregated into a single major theme of "Migrated `kubelet` to structured logging", so we might want to coordinate with the folks working on this migration on what would be the best summary for the changes.

Reply:
For structured log migration, the community has broken down into sub tasks and that is why we see the release notes in every commit related. Ref: https://github.com/kubernetes/kubernetes/issues/104197

@parul5sahoo Please make sure the phrase is consistent. I think we use `structured logging` more often in the past. Let's keep it this way.

Reply:
When it comes to Major Themes, we might be able to combine some of them together